src/CMakeLists.txt

```diff
@@ -312,7 +312,7 @@ merge_static_libraries(${core_library} "${src_combined_nupiccore_source_archives
 #
 # For Linux, OSx; the .so file will contain all symbols.
 #
-if(MSVC)
+if(true)
```

Comment:
So, does this mean we are not doing the shared library any more?

Reply:
thanks! forgotten from a past debugging, I'll revert that